src/main/java/lv/acodemy/homework/PezDispenser.java

```diff
@@ -0,0 +1,113 @@
+package lv.acodemy.homework;
+
+import scala.collection.parallel.ParIterableLike;
+
+public class PezDispenser {
+
+    private String color;
+    private String pezName;
+    private String seriesName;
+    private int candyCount;
+    private final int MAX_CANDIES = 12;
+
+    public PezDispenser(String color, String pezName, String seriesName) {
+        this.color = color;
+        this.pezName = pezName;
+        this.seriesName = seriesName;
+        this.candyCount = MAX_CANDIES;
+    }
+    public String getColor() {
+        return color;
+    }
+    public String getPezName() {
+        return pezName;
+    }
+    public String getSeriesName() {
+        return seriesName;
+    }
+    public int getCandyCount() {
+        return candyCount;
+    }
+    public int getMax_Candies() {
+        return MAX_CANDIES;
+    }
+
+    // - отдавать по одной конфетке
+    public void eatCandy() {
+        if (candyCount == 0) {
+            System.out.println("Empty Pez Dispenser.");
+            System.out.println("Max is: 12. Current: " + candyCount);
+        } else {
+            System.out.println("Give me one candy!");
+            System.out.println("Max is: 12. Current: " + candyCount);
+            candyCount--;
+        }
+    }
+
+    // - отдавать любое количество нам нужных конфет
+    public void eatCandies(int enterCandyCount) {
+        if (enterCandyCount > MAX_CANDIES) {
+            System.out.println("You asking too much candies.");
+            System.out.println("Max is: 12. Current: " + candyCount);
+        } else if (enterCandyCount < 1) {
+            System.out.println("Impossible to take the candies.");
+            System.out.println("Max is: 12. Current: " + candyCount);
+        } else if (enterCandyCount > candyCount) {
+            System.out.println("Pez Dispenser does not have candies.");
```

Comment:
you can check if enterCandyCount > MAX_CANDIES, then print that it also impossible to take more candies than pez dispenser can handle